src/lib/common/HelpIcon.svelte

```diff
@@ -0,0 +1,7 @@
+<script lang="ts">
+  import icon from "../../../assets/help.svg";
+
+  export let contents: string;
+</script>
+
+<img src={icon} title={contents} alt={contents} />
```

Comment:
In Firefox, the tooltip takes a second to appear. And maybe the help icon looks clickable and it isn't, and that's an issue? Happy to do something better if there are specific ideas/examples